### PR TITLE
Added cooling and DHW power production/consumption 

### PR DESCRIPTION
**Pull request by arhein123**

### CODE REVIEW

File: heatpump.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 external_components:
-  - source: github://tobias-93/esphome-ecodan-heatpump@main
+  - source: github://arhein123/esphome-ecodan-heatpump@main
```
Comment (tobias-93) [inline heatpump.yaml:2]: Please revert

File: heatpump.yaml [hunk 2]
```diff
@@ -102,10 +102,19 @@ sensor:
...
-      name: Energy consumption (yesterday)
+      name: Heating Energy consumption (yesterday)
     energy_prod_yesterday:
+      name: Heating Energy production (yesterday)
+    energy_cooling_cons_yesterday:
+      name: Cooling Energy consumption (yesterday)
+    energy_cooling_prod_yesterday:
+      name: Cooling Energy production (yesterday)
+    energy_dhw_cons_yesterday:
+      name: Energy consumption (yesterday)
```
Comment (tobias-93) [inline heatpump.yaml:113]:
```suggestion
      name: Hot water energy consumption (yesterday)
```

File: heatpump.yaml [hunk 3]
```diff
@@ -102,10 +102,19 @@ sensor:
     water_flow:
       name: Water flow
     energy_cons_yesterday:
-      name: Energy consumption (yesterday)
+      name: Heating Energy consumption (yesterday)
```
Comment (tobias-93) [inline heatpump.yaml:105]:
```suggestion
      name: Heating energy consumption (yesterday)
```

File: heatpump.yaml [hunk 4]
```diff
@@ -102,10 +102,19 @@ sensor:
...
     energy_prod_yesterday:
+      name: Heating Energy production (yesterday)
```
Comment (tobias-93) [inline heatpump.yaml:107]:
```suggestion
      name: Heating energy production (yesterday)
```

File: heatpump.yaml [hunk 5]
```diff
@@ -102,10 +102,19 @@ sensor:
...
+    energy_cooling_cons_yesterday:
+      name: Cooling Energy consumption (yesterday)
```
Comment (tobias-93) [inline heatpump.yaml:109]:
```suggestion
      name: Cooling energy consumption (yesterday)
```

File: heatpump.yaml [hunk 6]
```diff
@@ -102,10 +102,19 @@ sensor:
...
+    energy_cooling_prod_yesterday:
+      name: Cooling Energy production (yesterday)
```
Comment (tobias-93) [inline heatpump.yaml:111]:
```suggestion
      name: Cooling energy production (yesterday)
```

File: heatpump.yaml [hunk 7]
```diff
@@ -102,10 +102,19 @@ sensor:
...
+    energy_dhw_cons_yesterday:
+      name: Energy consumption (yesterday)
+    energy_dhw_prod_yesterday:
       name: Energy production (yesterday)
```
Comment (tobias-93) [inline heatpump.yaml:115]:
```suggestion
      name: Hot water energy production (yesterday)
```

**Review by tobias-93 — CHANGES_REQUESTED** [review]
Thanks @arhein123! Nice to have these added fields. Please revert the repo name change and fix the casing of the sensor names, then I'm happy to merge 👍🏼

**Comment by arhein123** [comment]
I think i updated it with your suggestions. First time doing this.. ;)

**Review by tobias-93 — APPROVED** [review]

**Comment by tobias-93** [comment]
Looks good to me 👍🏼  Thanks!

**Pull request merged**